storage/statedb/cache_fastcache.go

```diff
@@ -41,19 +41,19 @@ type FastCache struct {
 }
 
 // NewFastCache creates a FastCache with given cache size.
-// If you want auto-scaled cache size, set cacheSizeMB to AutoScaling.
+// If you want auto-scaled cache size, set config.FastCacheSizeMB to AutoScaling.
 // It returns nil if the cache size is zero.
-func NewFastCache(cacheSizeMB int) TrieNodeCache {
-	if cacheSizeMB == AutoScaling {
-		cacheSizeMB = getTrieNodeCacheSizeMB()
+func NewFastCache(config TrieNodeCacheConfig) TrieNodeCache {
+	if config.FastCacheSizeMB == AutoScaling {
```

Comment:
This is not relevant to the pr, but I think `LocalCacheSizeMB` is better than `FastCacheSizeMB`.

Reply:
I think so too.

Reply:
Applied. Thanks!